packages/web-pkg/src/components/FilesList/ResourceLink.vue

```diff
@@ -90,15 +93,17 @@ export default {
     }
   },
   methods: {
-    emitClick() {
+    emitClick(e: unknown) {
+      if (!e || typeof (e as MouseEvent).stopPropagation !== 'function') {
```

Comment:
didn't checked the whole pr, but I would actually double check with mobile device bc we have different types of events here